index.js

```diff
@@ -152,7 +152,7 @@ function makeError(result, options) {
 		error = new Error(message);
 	}
 
-	const prefix = getErrorPrefix({timedOut, timeout, signal, exitCodeName, exitCode});
+	const prefix = getErrorPrefix({timedOut, timeout, signal, exitCode, exitCodeName, isCanceled});
```

Comment:
Don't do unrelated changes

Reply:
I might have done this while fixing some merge conflict; that's the only time I touched those two variables `exitCode` and `exitCodeName`